src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -276,7 +276,7 @@ protected virtual async Task<Stream> CloneRequestBodyAsync(HttpRequest request,
         }
         else
         {
-            Logger.LogWarning("Aggregation does not support body copy without Content-Length header!");
+            Logger.LogInformation("Aggregation does not support body copy without Content-Length header, skipping body copy.");

```

Comment:
I recommend implementing a minor decision-making process based on the HTTP verb
- If the verb is GET, then it should be logged at the `Information` level, as should HEAD, OPTIONS, and other verbs that do not have a body.
- However, if the verb is PUT, POST, etc., then we should log at the `Warning` level.

This approach should suffice to determine the appropriate logging level.

Reply:
Hi raman,

Hard disagree for me on this, as the presence or absence of body is not specified in the HTTP spec.

It's an entirely normal and supported scenario to have a PUT or POST query without body and it stills holds a meaning (https://lists.w3.org/Archives/Public/ietf-http-wg/2010JulSep/0276.html)

So I wouldn't log it as a warning and I wouldn't make a verb-specific logic here.

Reply:
Fair enough! Not an issue.